tuxi: add page (#5357)
tldr-pages/tldr: pages/linux/tuxi.md

# tuxi

> A CLI tool that scrapes Google search results and SERPs that provides instant and concise answers.
> More information: <https://github.com/Bugswriter/tuxi>.

> - Use Google to search for a string:

Reviewer: What if we start with a verb?
```suggestion
- Make a search using Google:
```
What do you think @bl-ue ?

> `tuxi {{search_string}}`

Reviewer: ```suggestion
`tuxi {{search_terms}}`
```

> - Show [r]aw search results (no pretty output, no colors):

Reviewer: Perhaps is this better:
```suggestion
- Display the search results in [r]aw format (no pretty output, no colors):
```

`tuxi -r {{search_string}}`

> - Show only search results (silences "Did you mean?", greetings and usage):

Reviewer: ```suggestion
- Display only search results (silences "Did you mean?", greetings and usage):
```

`tuxi -q {{search_string}}`

> - Display help and usage of `tuxi`:

Reviewer: I think `tuxi` is redundant in the example description.
```suggestion
- Display help:
```
